Library/Formula/h2.rb

```diff
@@ -17,6 +17,9 @@ def install
     # Remove windows files
     rm_f Dir["bin/*.bat"]
 
+    system "cat bin/h2.sh | col -b > bin/_tmp_h2.sh"
```

Comment:
Instead of shelling out, how about doing this in Ruby?

```ruby
script = File.read("bin/h2.sh").gsub("\r\n", "\n")
File.open("bin/h2.sh", "w") {|f| f.write script}
```